packages/app-page-builder/src/editor/components/DropZone/Horizontal.tsx

```diff
@@ -46,13 +47,21 @@ export interface HorizontalPropsType {
     onDrop: DroppableOnDropPropType;
     below?: boolean;
     isVisible?: DroppableIsVisiblePropType;
+
+    /**
+     * For testing purpose
+     */
+     "data-testid"?: string;
 }
 
-const HorizontalComponent: React.FC<HorizontalPropsType> = ({ below, onDrop, isVisible, type }) => {
+const HorizontalComponent: React.FC<HorizontalPropsType> = props => {
+    const { below, onDrop, isVisible, type } = props;
     return (
         <Droppable type={type} isVisible={isVisible} onDrop={onDrop}>
             {({ isOver, drop }) => (
                 <div
+                    className={props["data-testid"]}
```

Comment:
Do we need to pass the test ID to `className`? I don't think so, but correct me if I'm wrong.